docs/OV_Runtime_UG/deployment/deployment-manager-tool.md

```diff
@@ -1,10 +1,10 @@
 # Deployment Manager {#openvino_docs_install_guides_deployment_manager_tool}
 
-The Deployment Manager is a Python* command-line tool that creates a deployment package by assembling the model, IR files, your application, and associated dependencies into a runtime package for your target device. This tool is delivered within the Intel® Distribution of OpenVINO™ toolkit for Linux*, Windows* and macOS* release packages and is available after installation in the `<INSTALL_DIR>/tools/deployment_manager` directory.
+The Deployment Manager is a Python command-line tool that creates a deployment package by assembling the model, OpenVINO IR files, your application, and associated dependencies into a runtime package for your target device. This tool is delivered within the Intel® Distribution of OpenVINO™ Toolkit for Linux, Windows and macOS release packages.  It is available after the installation in the `<INSTALL_DIR>/tools/deployment_manager` directory.
 
 ## Prerequisites
 
-* Intel® Distribution of OpenVINO™ toolkit
+* Intel® Distribution of OpenVINO™ Toolkit
 * To run inference on a target device other than CPU, device drivers must be pre-installed:
    * **For Linux**, see the following sections in the [installation instructions for Linux](../../install_guides/installing-openvino-linux.md):
```

Comment:
```suggestion
   * **For Linux**, see the following sections:
```

Reply:
These referenced pages are identical. Let's use hardware device for grouping instead of operating system. So there would be only 4 items in the list: For GPU/VPU/GNA/NCS2.